### DISCUSSION TITLE
How do I turn off icon preview(请问如何关闭图标预览)

### DISCUSSION THREAD

**Discussion by Listen-to-poetry-in-the-wind** [unocss/unocss#4313] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/de51e087-4cdc-44d8-a11c-9171f848e450)

**Comment by ntnyq** [comment, marked as answer]
Do you use `Iconify IntelliSense`, just set `iconify.annotations` to false.

**Reply by Listen-to-poetry-in-the-wind** [reply]
thank you
